### PR TITLE
Packer Community Plugins 0.0.1 (new formula).

### PR DESCRIPTION
**Pull request by mefellows**
Created a new Formula for the Packer Community Plugins project. This plugin
depends on Packer itself and performs a simple template validation
using upstream Packer and one of the Community plugins.

### CODE REVIEW

File: Library/Formula/packer-community-plugins.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class PackerCommunityPlugins < Formula
+  homepage "https://github.com/packer-community/packer-windows-plugins"
+  version "0.0.1"
+
+  if Hardware.is_64_bit?
+    url "https://github.com/packer-community/packer-windows-plugins/releases/download/pre-release/darwin_amd64.zip"
```
Comment (mistydemeo) [inline Library/Formula/packer-community-plugins.rb]: Can this be built from source?

Comment (mistydemeo) [inline Library/Formula/packer-community-plugins.rb]: Is this software stable yet? It looks like `pre-release` is the only version so far. Where did you get the 0.0.1 number?

**Comment by mefellows** [comment]
Hi @mistydemeo, thanks for your prompt response!

WRT building from source, I've really just followed the way the main [packer](https://github.com/Homebrew/homebrew-binary/blob/master/packer.rb) project was bundled up and adapted for this project. We could build from source if that is really preferred?

In terms of versioning, I was probably a bit over-excited about this, really it should have been 0.0.1-pre-release but a sane versioning strategy has not yet been [agreed](https://github.com/packer-community/packer-windows-plugins/issues/20) upon. For now, I will close this PR until we have clarity on this.

**Comment by mistydemeo** [comment]
Thanks - waiting sounds like the best strategy.

Packer itself is in the [homebrew-binary](https://github.com/Homebrew/homebrew-binary/) repository, not this one, since it's not built from source. (Preferably it also should get migrated to a source build at some point.)

**Comment by mefellows** [comment]
OK will do, thanks.

When I resubmit I'll port it to a CLI build and then will do the same for Packer itself.

**Pull request closed without merging**